docs/apache-airflow/concepts/dags.rst

```diff
@@ -745,3 +745,34 @@ the dependency graph.
 
 The dependency detector is configurable, so you can implement your own logic different than the defaults in
 :class:`~airflow.serialization.serialized_objects.DependencyDetector`
+
+DAG pausing, deactivation and deletion
+--------------------------------------
+
+The DAGs have several states when it comes to being "not running". DAGs can be paused, deactivated
+and finally all metadata for the ``DAG`` can be deleted.
+
+Dag can be paused via UI when it is present in the ``DAGS_FOLDER``, and scheduler stored it in
+the database, but the user chose to disable it via the UI. The ``pause/unpause`` actions are available
+via UI and API. Paused ``DAG`` is not scheduled by the Scheduler, but you can trigger them via UI for
+manual runs.
```

Comment:
This is a bit confusing I think?
When we trigger a manual RUN it also automatically change the paused DAG to ON so it can run. If it is left in Off the dag will not be scheduled.

Reply:
I am not sure if this is even "good" - I checked that his is the behaviour in current Airflow ("development") but maybe we should actually fix it :) ?

Reply:
Actually I got it wrong :).. This is really misleading for new people. I will correct it :)